test/TensorFlowNET.Keras.UnitTest/SaveModel/SequentialModelSave.cs

```diff
@@ -18,15 +18,15 @@ public class SequentialModelSave
     [TestMethod]
     public void SimpleModelFromAutoCompile()
     {
-        var inputs = tf.keras.layers.Input((28, 28, 1));
-        var x = tf.keras.layers.Flatten().Apply(inputs);
-        x = tf.keras.layers.Dense(100, activation: tf.nn.relu).Apply(x);
-        x = tf.keras.layers.Dense(units: 10).Apply(x);
-        var outputs = tf.keras.layers.Softmax(axis: 1).Apply(x);
-        var model = tf.keras.Model(inputs, outputs);
+        var inputs = keras.layers.Input((28, 28, 1));
```

Comment:
The official example use `tf.keras.xxx` more.

Reply:
OK, I'll revise it.